client/stores/useAuth.ts

```diff
@@ -27,23 +27,30 @@ export const useAuth = s<UseAuthComposer>(defineStore('auth', (): UseAuthCompose
   }
 
   async function login(values: Record<string, any>) {
-    const { data, error } = await executeLogin({ data: values })
-    if (!error.value) {
-      setTokens(data.value)
-      loggedInEvent.emit({ tokens: data.value })
+    let tokens: Tokens
+    try {
+      const { data } = await axios().post<Tokens>('/login', values)
+      tokens = data
     }
-    else {
-      loggedInEvent.emit({ error: error.value })
+    catch (error) {
+      loggedInEvent.emit({ error: error as AxiosError })
+      return
     }
+
+    setTokens(tokens)
+    loggedInEvent.emit({ tokens })
   }
 
   async function logout() {
-    const { error } = await executeLogout()
-    if (error.value)
+    try {
+      await axios().post('/logout')
+    }
+    catch (error) {
+      loggedOutEvent.emit({ error: error as AxiosError })
       return
+    }
 
     setTokens(null)
-    navigateTo('/login')
     loggedOutEvent.emit()
   }
```

Comment:
reorganize the try catch as we discussed earlier. It's more idiomatic, and cleaner in the end, in my opinion

that is to say, catch everything, but guard on the type (to catch AxiosError) and rethrow anything else. This will also avoid the need for the unsafe cast, as an added benefit.